frontend/components/home/pug/home.pug

```diff
@@ -23,7 +23,7 @@ div.homeContainer(
 
 		div.homeTopBar
 		#homeLogo(style="pointer-events:{{vm.pointerEvents}}" aria-label="Home")
-			img(ng-src="{{vm.topLogo}}", alt="3D Repo Logo (White)" ng-click="vm.home()")
+			img(ng-src="{{vm.topLogo}}" alt="3D Repo" longdesc="3DRepoBuildingInformationModellingSofwtare" ng-click="vm.home()")
```

Comment:
Software is spelt wrong. Also, why are there no spaces?

Reply:
Hi Charence, 
 It turns out the "longdesc" attribute shouldn't have any spaces in it i thought it was a bit odd also